### ISSUE TITLE
Стена

### ISSUE THREAD

**Issue by rsking01** [Defasium/vkCaptchaBreaker#21]
Доброго времени суток. Почему капча на стенах не разгадывается? Я 100 раз пытался опубликовать пост, не одна из 100 капч не была решена.

**Comment by miztizm** [comment]
То есть мобильная версия? на m.vk.com?

**Comment by rsking01** [comment]
> То есть мобильная версия? на m.vk.com?

Да. Когда во время публикации поста вылазит капча, он ее не может решить. В остальных местах все решает. Мне кажется это происходит из-за того, что в капче на стене символов больше, чем в капче, которая к примеру в диалогах.
